crates/sui-verifier/src/entry_points_verifier.rs

```diff
@@ -187,6 +187,9 @@ fn verify_param_type(
     }
 
     if is_object(view, function_type_args, param)? {
+        return Ok(());
+    }
+    if is_object_vector(view, function_type_args, param)? {
```

Comment:
Took me a second staring at this. Maybe just `if is_object(view, function_type_args, param)? || is_object_vector(view, function_type_args, param)?` ?

I generally really like the if-return style of guards. Just confused me as its immediately followed by an if that "returns"